test/functional/lua/ui_event_spec.lua

```diff
@@ -42,7 +42,6 @@ describe('vim.ui_attach', function()
   end
 
   it('can receive popupmenu events', function()
-    exec_lua [[ vim.ui_attach(ns, {ext_popupmenu=true}, on_event) ]]
```

Comment:
don't need to change the test, this is to show the baseline before invoking `complete()`